src/MuxManager.cpp

```diff
@@ -570,6 +570,9 @@ void MuxManager::handleWarmRestartReconciliationTimeout(const boost::system::err
         MUXLOGWARNING("Reconciliation timed out after warm restart, set service to reconciled now.");
     }
 
+    uint32_t rc = system("sudo config muxcable mode auto all"); // rc > 0 means success
```

Comment:
I think we can add a UT to ensure when reconciliation is completed the config is indeed set to auto?

Reply:
It's a bit hard, `linkmgrd` UTs doesn't use mock DBs.